### ISSUE TITLE
add a component that can get the attached users online status

### ISSUE THREAD

**Issue by Kirv-goldblade** [Yellow-Dog-Man/Resonite-Issues#1050]
### Is your feature request related to a problem? Please describe.

currently the online/away/busy status is only useful in the contact menu and can't be seen in world in anyway

### Describe the solution you'd like

i'd like for a component or flux node that can check the user's status and provide an output that can be checked in some way (either as an index or even just a set of boolean values that would allow the user to sent up an avatars side busy indicator or proxy to trigger the easy away for if they need to go AFK without leaving the world they are in. 

### Describe alternatives you've considered

i suspect it would be possible to setup a facet to pull this data from the userspace and send it to a variable in the world but that would restrict any functions VR users only.

**Comment by Frooxius** [comment]
Hmm, this might be added, but there is one thing to be vary of - this can potentially reveal information that user does not want revealed.

If it were added, we'd only make the user that it's for actually update it, so you can't use it to fetch the online status of someone else.

Problem is, that some users might not want to have their non-contacts know their online status, even if they are present in the world, which might outweigh adding this as an official thing, because it could be abused for that.

**Comment by Kirv-goldblade** [comment]
oh yeah 100% i had intended this to only be able to function if it was part of an active avatar. so it would only function if you where wearing/holding the avatar/object, maybe you could make it work like the simple avatar protection so the user to be affected has to be the one to create the component

**Comment by ShrikeAlvaron** [comment]
Presumably Froox's idea for the node is that it would output the local user's status, however the concern is that in a world with other people it's trivial to get a pulse from another user in order to write their status to a variable that you can read.

**Comment by Frooxius** [comment]
The main point is that it would only work when the user is present in the world.

If it only works under the avatar and such, people will find ways to sneak that in under the user and get their status, even if the user hasn't put it there in the first place.

This is where a custom solution with cloud variable might be better - instead of having a single standardized system, each user has their own solution, which makes abuse harder.

**Comment by Kirv-goldblade** [comment]
yeah, the big thing for me was needing a way to get the status in-order to hand it off to the actual functions i wanted to build but in all my attempts to find it pointed to it being stored in the "user" space with no way to get it into the "world" or "avatar" spaced

**Comment by Frooxius** [comment]
You can proxy anything from userspace to worldspace using cloud variables.

**Comment by Kirv-goldblade** [comment]
oh, guess i just need to find out what the variable is that actually stores my online status then but no-one seems to know (otherwise i suspect what i'm trying to create would have already been made

**Comment by Frooxius** [comment]
There's not one. You need to make one and then fill it with your online status in userspace and read that variable in worldspace.

It's a general mechanism that you can use to manually get data into worldspace that you want there.

Some of the data, we will not automatically expose, because then anyone can access it, without the user having a choice. The cloud variables let you bypass that, but it's essentially you making that choice at that point.

**Comment by Kirv-goldblade** [comment]
so basicaly what i'm hearing i could create a facet that pulls my status from my userspace and translated it to a cloud variable, then use that varable to create like a online/away/busy indicator for my avatar. guess i'll have to way till desktop users can UI edit to proceed further
